src/UserStories/UserStories.css

```
@@ -125,8 +135,24 @@ html {
 .story-actions {
   justify-self: flex-end;
 }
-.story-actions button {
+.story-action {
+  padding: 0;
+  background: none;
+  border: none;
   color: white;
+  cursor: pointer;
+}
+.story-action + .story-action {
```

Add a comment before this because it's not obvious.
```suggestion
// only add margin between two adjacent items
.story-action + .story-action {
```

---

Done

---

The margin-left: 5px is clear, I mean the comment above the selector (because the selector is the unclear part).